### PR TITLE
[Improvement-5918]Remove the code logger.isDebugEnabled()

### PR DESCRIPTION
**Pull request by fbjoker**
## Purpose of the pull request
Remove the code logger.isDebugEnabled() in  module  reomte

## Brief change log
remove the code logger.isDebugEnabled() 

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Comment by DarkAssassinator** [comment]
same as PR:https://github.com/apache/dolphinscheduler/pull/12389 ?

**Comment by fbjoker** [comment]
I don't know how to pass milestone-label-check

**Comment by fuchanghai** [comment]
> same as PR:#12389 ?

it looks like same as this PR

**Comment by fbjoker** [comment]
yes,  I don't know how to pass milestone-label-check, so  I modified commite meage and resubmitted it，bug its also  check fail. Can I cancel the submission

**Comment by DarkAssassinator** [comment]
> yes, I don't know how to pass milestone-label-check, so I modified commite meage and resubmitted it，bug its also check fail. Can I cancel the submission

Sure, u can close one. And commiter will add mileston-label.

**Comment by fbjoker** [comment]
> > yes, I don't know how to pass milestone-label-check, so I modified commite meage and resubmitted it，bug its also check fail. Can I cancel the submission
> 
> Sure, u can close one. And commiter will add mileston-label.

thanks, i close PR #12389

**Comment by fuchanghai** [comment]
please relation with issue that need be closed

File: dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/handler/NettyClientHandler.java [hunk 1]
```diff
@@ -187,9 +187,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
             heartBeat.setBody(heartBeatData);
             ctx.channel().writeAndFlush(heartBeat)
                     .addListener(ChannelFutureListener.CLOSE_ON_FAILURE);
-            if (logger.isDebugEnabled()) {
-                logger.debug("Client send heart beat to: {}", ChannelUtils.getRemoteAddress(ctx.channel()));
-            }
+            logger.debug("Client send heart beat to: {}", ChannelUtils.getRemoteAddress(ctx.channel()));
```
Comment (ruanwenjun) [inline dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/handler/NettyClientHandler.java:190]: In fact, I don't agree with remove `logger.isDebugEnabled` here, use `logger.debug` will need to calculate `ChannelUtils.getRemoteAddress(ctx.channel())` in each log, but in most of times we don't open debug, we don't need to calculate this.

**Comment by ruanwenjun** [comment]
I don't think this PR is needed, if no other concerns, I will close this PR.

**Pull request closed without merging**